CHANGELOG.md

```diff
@@ -21,6 +21,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
  - #2283, Fix infinite recursion when loading schema cache with self-referencing view - @wolfgangwalther
  - #2343, Return status code 200 for PATCH requests which don't affect any rows - @wolfgangwalther
  - #2481, Treat computed relationships not marked SETOF as M2O/O2O relationship - @wolfgangwalther
+ - #2362, #2475, #2537, Stricter parsing of query string. Instead of silently ignoring, the parser now throws on invalid syntax like json paths for embeddings, hints for regular columns, empty casts or fts languages, etc. - @wolfgangwalther
```

Comment:
Love :heart: this change. Should we also include an entry on `### Changed` for this?

Reply:
I mean, some requests could start failing for some users. I got the same effect when doing https://github.com/PostgREST/postgrest/issues/1771.